### PR TITLE
fixed CQ for bitmessageqt.statusbar module

### PR DESCRIPTION
**Pull request by coolguy-cell**
flake8 fixes
pycodestyle fixes
pylint fixes

### CODE REVIEW

File: src/bitmessageqt/statusbar.py [hunk 1]
```diff
@@ -32,7 +37,9 @@ def addImportant(self, message):
         self.timerEvent(None)
 
     def showMessage(self, message, timeout=0):
```
Comment (PeterSurda) [inline src/bitmessageqt/statusbar.py:39]: this method can probably be removed

File: src/bitmessageqt/statusbar.py [hunk 2]
```diff
@@ -32,7 +37,9 @@ def addImportant(self, message):
...
+        """It will show the message"""
         super(BMStatusBar, self).showMessage(message, timeout)
 
     def clearMessage(self):
+        """It clear the message"""
```
Comment (PeterSurda) [inline src/bitmessageqt/statusbar.py:44]: can also be removed

File: src/bitmessageqt/statusbar.py [hunk 3]
```diff
@@ -13,6 +17,7 @@ def __init__(self, parent=None):
         self.iterator = 0
 
     def timerEvent(self, event):
+        """It indicates timer event."""
```
Comment (PeterSurda) [inline src/bitmessageqt/statusbar.py:20]: this is an event handler, not an indicator. It allows to queue and prioritise messages to show in the status bar, for example if many messages come very quickly after one another, it adds delays and so on

File: src/bitmessageqt/statusbar.py [hunk 4]
```diff
@@ -1,8 +1,12 @@
-from PyQt4 import QtCore, QtGui
-from Queue import Queue
+# pylint: disable=useless-super-delegation,unused-argument
```
Comment (PeterSurda) [inline src/bitmessageqt/statusbar.py:1]: after removing the two methods, `useless-super-delegation` can be removed too

**Comment by coolguy-cell** [comment]
Done.

File: src/bitmessageqt/statusbar.py [hunk 5]
```diff
@@ -13,6 +17,9 @@ def __init__(self, parent=None):
         self.iterator = 0
 
     def timerEvent(self, event):
+        """It is an event handler which allows to queue and prioritise messages to
```
Comment (PeterSurda) [inline src/bitmessageqt/statusbar.py:20]: no "It is"

File: src/bitmessageqt/statusbar.py [hunk 6]
```diff
@@ -1,8 +1,12 @@
...
-from Queue import Queue
+# pylint: disable=unused-argument
+"""Status bar Module"""
+
 from time import time
+from PyQt4 import QtGui
+
 
 class BMStatusBar(QtGui.QStatusBar):
+    """BMStatusBar Class for Qt"""
```
Comment (PeterSurda) [inline src/bitmessageqt/statusbar.py:9]: `"""Status bar with queue and priorities"""`

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**